### DISCUSSION TITLE
Constraints and technologies

### DISCUSSION THREAD

**Discussion by amelield** [unoplatform/uno#10332] [Ideas]
I think the main goal would be to create a test, a diagnostic tool or at least a static chart that would consider the constraint or possibilities to do something in a Uno project according to the technologies used. I am not sure where to start. Is there some documentation on some of these constraints already? At which scale would it be helpful? I will start the search for documentation and tools and will try to update it here. I would welcome any feedback or lead.  @jeromelaban I would be grateful if you could give me a summary of what you had in mind.

**Comment by jeromelaban** [comment]
Do you have a specific scenario in mind? We currently [have this](https://platform.uno/docs/articles/implemented-views.html), but it's very specific to each control.

**Reply by amelield** [reply]
Maybe something like that with the implication of implemented properties and methods.
An other example of limitation is svg rasterization is not available in Wasm. An other is RawBitmap renderer cis not available in Wasm so some tests cannot be run in runtime. Are there some limitations in developping for xamarin? The most important known limitations could be available at the same place or using the same tool, if any of this make sense.
